code/modules/mob/living/carbon/human/species/station/slime.dm

```diff
@@ -0,0 +1,88 @@
+
+/datum/species/slime
+	name = "Slime"
+	name_plural = "slimes"
+
+//	spawn_flags = CAN_JOIN
+
+	default_form = FORM_SLIME
+	obligate_form = TRUE
+	reagent_tag = IS_SLIME
+	unarmed_types = list(/datum/unarmed_attack/slime_glomp)
+	flags = NO_SLIP | NO_BREATHE | NO_BLOOD | NO_SCAN | NO_MINOR_CUT
+	siemens_coefficient = 3 //conductive
+	darksight = 3
+	virus_immune = TRUE
+	breath_type = null
+	poison_type = null
+	hunger_factor = DEFAULT_HUNGER_FACTOR * 2
+
+	burn_mod = 1
+	brute_mod = 1
+	toxins_mod = -1 // Flips the affects of toxins, so dealing toxins heals well reducing toxins damages - R4D6|Trilby
+	oxy_mod = 0
+
+	has_process = list(
+		BP_BRAIN = /obj/item/organ/internal/brain/slime
+		)
+
+	breath_type = null
+	poison_type = null
+
+	bump_flag = SLIME
+	swap_flags = MONKEY|SLIME|SIMPLE_ANIMAL
+	push_flags = MONKEY|SLIME|SIMPLE_ANIMAL
+
+	has_process = list(
+		OP_HEART   = /obj/item/organ/internal/heart/slime,
+		BP_BRAIN   = /obj/item/organ/internal/brain/slime,
+		OP_EYES    = /obj/item/organ/internal/eyes/slime,
+		OP_STOMACH = /obj/item/organ/internal/stomach/slime
+		)
```

Comment:
This list is already defined a few lines above.
And I don't think they need the eye organs to see, or else I would have noticed during testing.
And make the heart & stomach part of the core instead of them being separate organs if possible, because the core is supposed to be the _**only**_ organ that slime-people require.